source/blog/2018-11-08-a-new-home-for-tutorials.md

```diff
@@ -0,0 +1,8 @@
+---
+title: A new home for Tutorials
+author: Learning Core Team
+tags: Recent Posts
+responsive: true
+---
+
+TBK
```

Comment:
I like the idea, but the implementation seems ... _lacking_? 😉

Reply:
@locks what do you want the tutorial announcement to be?

Reply:
I want to explain the decision of moving the tutorials out of the guides and making them non-versioned, and give a tiny glimpse of our roadmap if the team agrees.